src/tiny-slider.js

```diff
@@ -876,7 +876,11 @@ export var tns = function (options) {
     if (horizontal && !isServer) {
 
       if (PERCENTAGELAYOUT || autoWidth) {
-        addCSSRule(sheet, '#' + slideId + ' > .tns-item', 'font-size:' + win.getComputedStyle(slideItems[0]).fontSize + ';', getCssRulesLength(sheet));
+        let fontSize = win.getComputedStyle(slideItems[0]).fontSize;
+        if (fontSize == undefined || fontSize == '') {
+          fontSize = 0;
```

Comment:
not sure if it would be better to do:
```
          fontSize = 'inherit';
```

I tested both on my getty-ui branch and had no issues.  Thoughts?